modules/representation_management/app/controllers/representation_management/v0/pdf_generator_2122a_controller.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module RepresentationManagement
+  module V0
+    class PdfGenerator2122aController < RepresentationManagement::V0::PdfGeneratorBaseController
+      # service_tag 'lighthouse-veteran' # Is this the correct service tag?
+
+      def create
+        # We'll need a process here to check the params to make sure all the
+        # required fields are present. If not, we'll need to return an error
+        # with status: :unprocessable_entity.  If all fields are accounted for
+        # we need to fill out the 2122 PDF with the data and return the file
+        # to the front end.
+
+        # This work probably belongs in the PDF Generation ticket.
+        render json: {}, status: :unprocessable_entity
+      end
+
+      private
+
+      def form_params
```

Comment:
You'll have to params.require the non-optional params.

Reply:
I can require the top level objects here (`veteran`, `claimant`, `service_organization`, etc) without any trouble.  The issue is requiring nested values within those top level objects.  I know we discussed returning a `400` response when required parameters weren't present.  Right now I can require a top level object and lean on rails presence validations to require nested values, but a missing required nested value won't return a `400`, it will return a `422` for the validation error.

Does that work in this case?